packages/broadway.rb

```diff
@@ -0,0 +1,77 @@
+require 'package'
+
+class Broadway < Package
+  description 'Run GTK applications in a browser window.'
+  homepage 'https://developer.gnome.org/gtk3/stable/gtk-broadway.html'
```

Comment:
This is the homepage, not the source url.

Reply:
Fair enough.